### ISSUE TITLE
Add a GUI way to change username

### ISSUE THREAD

**Issue by marmarta** [QubesOS/qubes-issues#7749]
### The problem you're addressing (if any)

There is not GUI way of changing the username.

### The solution you'd like

Add it to new Global Settings.

### The value to a user, and who that user might be

People who prefer GUI tools (27% of our users, based on Survey) or like to use GUI tools not just CLI (65% of users).

**Comment by ben-grande** [comment]
@marmarta I think this is an upstream issue, not ours.

**Comment by marmarta** [comment]
> [@marmarta](https://github.com/marmarta) I think this is an upstream issue, not ours.

why? we have a lot of VMs, telling anyone "change all usersnames manually" sounds like a nightmare...

**Comment by ben-grande** [comment]
> > [@marmarta](https://github.com/marmarta) I think this is an upstream issue, not ours.
> 
> why? we have a lot of VMs, telling anyone "change all usersnames manually" sounds like a nightmare...

Maybe I don't understand the issue at all, because I don't even change dom0 user name from "user" or from any other qube. Qubes is also not a multi-user system, dom0 is not secured against that, so doesn't make sense to change on dom0. For privilege separation using GUI domains, it makes sense to have separate GUI domains instead of separate users on the same GUI domain.

Can you please specify the use case and benefit of changing user name?

**Comment by marmarta** [comment]
When I log into my system, I use a username, right? I set this username at some point in my life, and maybe I want to change it because:
- I changed my name
- I picked it while drunk at 3am and it's embarassing
- I picked it assuming I'd never have to go to office and yet home office is gone and I probably should have FuckMyBoss as my username

**Comment by andrewdavidwong** [comment]
> why? we have a lot of VMs, telling anyone "change all usersnames manually" sounds like a nightmare...

Wait, but the dom0 username only affects dom0, whereas the username for all other VMs is "user," right?

**Comment by marmarta** [comment]
> Wait, but the dom0 username only affects dom0, whereas the username for all other VMs is "user," right?

yeah, I think I was confused when I wrote the first answer, sorry. 

The general username I was referring to in this ticket is the thing you use to log into the system, and I'd like a gui for that.
